setup_template.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin python3
+
+import os
+import pathlib as pl
+
```

Comment:
Can you add `REPO_DIR = pl.Path(__file__).parent`? This can then be used to ensure that the other commands will work independent of the current working directory.

Reply:
I did `os.chdir(pl.Path(__file__).parent)` now to ensure working dir == repo root

Reply:
Out of curiousity, why do you prefer this approach? Seems to me like it introduces (possibly unexpected) side-effects for users that are harder to guard against.

In this case though, we should store the original directory and change it back in a try/finally to ensure that this doesn't change the users current directory. Otherwise things can get weird if anyone integrates this in another script.

Reply:
Oh it was just to keep the complexity low (no need to keep track of another variable) again because this is really only meant to be run once and then destroy itself. But either way is fine for me. 

In a "normal" program I would advise against ever changing the working directory, as it is nothing else but a hard to debug global variable.

Reply:
Would the working directory change for the parent process? Or do you mean if somebody imports this as a module and then calls main() from the module path. Maybe we should safeguard against that but it seems a bit overkill

Reply:
But yeah I'll change it to use REPO_DIR

Reply:
Huh thats my bad actually; I assumed it'd change it in the caller but it doesn't. This is why I get for being used to bash scripts for these kind of things. So yeah it doesn't actually matter.

Reply:
Done and tested